addons/mail/static/src/components/composer_text_input/composer_text_input.js

```diff
@@ -210,6 +210,7 @@ export class ComposerTextInput extends Component {
         }
         this._textareaLastInputValue = this._textareaRef.el.value;
         this._updateHeight();
+        this.composer.getLinkPreview();
```

Comment:
I think we need to smart-throttle this. If the previous "check" is not done, we shouldn't start a new one at every key press.